index.js

```diff
@@ -45,6 +47,13 @@ class TemplateCompiler extends Filter {
     return __dirname;
   }
 
+  registeredASTPlugins() {
+    // This is a super obtuse way to get access to the plugins we've registered
+    // it also returns other plugins that are registered by ember itself.
```

Comment:
Do we care about the ones in Ember itself (since they will never provide custom dependencies)?

Reply:
> Do we care about the ones in Ember itself

They are filtered out because they don't provide a `dependencies` method.

> since they will never provide custom dependencies

Is that a safe and/or necessary assumption?

Reply:
I mostly just don't like having to embed the assumption that we can read them in this way. Specifically, I am still planning on landing https://github.com/ember-cli/ember-cli-htmlbars/pull/130 and also converting Ember itself to actually _use_ the same registry as normal addon plugins.

Reply:
> I mostly just don't like having to embed the assumption that we can read them in this way. Specifically, I am still planning on landing #130 and also converting Ember itself to actually use the same registry as normal addon plugins.

I'm not sure how to turn this feedback into something actionable. Is there an alternative solution you can recommend or can you be more specific about your concerns?

Reply:
Don't worry about it for now, I'm assuming that #130 will land after this and I'll address my concerns there (and in ember itself).